cmd/ocsp-responder/main.go

```diff
@@ -123,6 +104,10 @@ type Config struct {
 		// LogSampleRate sets how frequently error logs should be emitted. This
 		// avoids flooding the logs during outages. 1 out of N log lines will be emitted.
 		LogSampleRate int `validate:"min=0"`
+
+		// Deprecated: ExpectedFreshness is no longer used now that we do not read
```

Comment:
Style nit. We typically use `DEPRECATED` below the existing comment.

Reply:
Yes, but we need to switch to `Deprecated:` because the godoc facilities that format deprecated fields differently don't correctly trigger on `DEPRECATED`. See, for example, https://pkg.go.dev/github.com/letsencrypt/boulder@v0.0.0-20230327182618-235d177a20f0/issuance#Certificate.ID